uiowa/uiowa
# Timeline media errors

tracks https://github.com/uiowa/uiowa/issues/4691

## To test:

1. pull branch
2. composer install
3. sync `stanleymuseum.uiowa.edu`
4. uli in to stanley
5. navigate to `/history`
6. edit the layout and check the timeline, you should see some `Media error` pills.
7. Updating the timeline items will error, and the top of the block will tell you what items are erroring.
8. Fix the `Media error` items by adding and removing an image
9. you should now be able to update without any errors.
